python/pyspark/sql/streaming/StateMessage.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package pyspark.sql.streaming;
+
+message StateRequest {
```

Comment:
is it possible to add some high level comments here or in some other Python file ?